### DISCUSSION TITLE
Should we have a regular meeting?

### DISCUSSION THREAD

**Discussion by westonpace** [substrait-io/substrait#170] [Polls]
I'd be happy to organize a regular (biweekly?) meeting (by organize I mean send out calendar invites / reminders and create Google Meet links).  I think we are starting to get to the point where there are a number of people working on different tasks in the community and it might be nice to have a regular sort of sync-up.

**Comment by westonpace** [comment]
Based on the feedback it seems there is some interest.  I would like to propose a (initially) bi-weekly sync call.  If you are interested then please update your availability here: https://www.when2meet.com/?15561603-Mpwvt

This first call is based around US timezones but I think we should do what other projects have done and alternate time zones between each meeting.  Feel free to make additional suggestions.

I have created a Google Doc for meeting minutes.  Feel free to add items to the agenda if you have anything you are interested in discussing: https://docs.google.com/document/d/1Afg93ojsVWdwo3rBO2Dtng8__RYqAy_zJMrnqhTQapc/edit?usp=sharing

**Comment by westonpace** [comment]
Based on the survey times I propose an initial meeting time of 8AM Pacific time on Wednesdays for sync meetings.

I have created a Google calendar for tracking the sync meetings:
https://calendar.google.com/calendar/embed?src=9lu5hflh1ggjkbko0hbomfte4k%40group.calendar.google.com&ctz=Pacific%2FHonolulu

ical: https://calendar.google.com/calendar/ical/9lu5hflh1ggjkbko0hbomfte4k%40group.calendar.google.com/public/basic.ics

**Comment by thisisnic** [comment]
The recording from today's sync call can be downloaded here: https://drive.google.com/file/d/1PJVk8Dya7aU49C4tRvlfkZA6JEFmoeKU/view?usp=sharing
